### DISCUSSION TITLE
Taskbar height and icon size 1.2.10 with 2024-08 Cumulative Update for Windows 11 23H2 - most of taskbar off bottom of screen

### DISCUSSION THREAD

**Discussion by Growlertiger** [ramensoftware/windhawk-mods#860] [General]
1.2.9 worked perfectly.
But today I updated to 1.2.10 and also installed the 2024-08 Cumulative Update for Windows 11 23H2.
Now it doesn't work correctly. The bottom half of the toolbar is off the bottom of the screen. Then even if I disable the mod I have to reboot to get Windows working normally.

**Comment by m417z** [comment]
Please see the following issue: #859.
